playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss

```diff
@@ -211,5 +212,9 @@
       }
     }
   }
+  //this margin is to allow typeahead error state to match text input error state
+  .typeahead-error-caption {
+    margin-top: -12px;
```

Comment:
Any reason we can't use padding instead of margin? And also, use a token of ours, even if it requires calculating with math a little.

Reply:
Margin is much more dangerous and can affect entire page layouts, especially negative margin.

Reply:
I wasn't 100% on this solution when I implemented it so would love feedback on what you think  a better way to handle this would be? Currently, the padding around the text input in 16px and is coming the textInput kit and if we just place the error message below it, there seems to be too much padding between the text input and the error. Rex wanted the error state to look similar to the error for texti nput and select where the padding between the input and error is 4px.